src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinMinecraft_ClearEffectRenderer.java

```diff
@@ -0,0 +1,28 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.multiplayer.WorldClient;
+import net.minecraft.client.particle.EffectRenderer;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(Minecraft.class)
+public abstract class MixinMinecraft_ClearEffectRenderer {
+
+    @Shadow
+    public EffectRenderer effectRenderer;
+
+    @Inject(
+            method = "loadWorld(Lnet/minecraft/client/multiplayer/WorldClient;Ljava/lang/String;)V",
+            at = @At(value = "HEAD"))
```

Comment:
can you inject right before it assigns the `this.theWorld = worldClientIn;` and not at the head of the method please